devops/ARC_README.md

```diff
@@ -0,0 +1,67 @@
+# Standing up ARC from scratch
```

Comment:
Q: What does ARC stand for? Might be helpful to mention it once.

Reply:
[Actions Runner Controller](https://github.com/actions/actions-runner-controller). Added the info and link as an intro paragraph to the document.